src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1179,6 +1286,210 @@ static void uct_ib_mlx5_devx_md_cleanup(uct_ib_md_t *ibmd)
     ucs_recursive_spinlock_destroy(&md->dbrec_lock);
 }
 
+#if HAVE_DECL_MLX5DV_DEVX_UMEM_REG_EX
+
+static const char *uct_ib_mkey_token = "uct_ib_mkey_token";
+
+static ucs_status_t
+uct_ib_mlx5_devx_reg_exported_key(uct_ib_md_t *ib_md, uint64_t flags,
+                                  void *address, size_t length,
+                                  uct_ib_mem_t *ib_memh)
+{
+    uct_ib_mlx5_md_t *md                                              =
+            ucs_derived_of(ib_md, uct_ib_mlx5_md_t);
+    uct_ib_mlx5_mem_t *memh                                           =
+            ucs_derived_of(ib_memh, uct_ib_mlx5_mem_t);
```

Comment:
minor: imo better leave these 2 lines unaligned (hard to read this way)

Reply:
done